.github/workflows/dev-release.yml

```diff
@@ -0,0 +1,48 @@
+name: Release @dev tag to npm
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  release:
+    if: ${{ github.event.label.name == 'release dev tag' }}
+    runs-on: ubuntu-22.04
+    env:
+      HUSKY: 0
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Setup PNPM
+        uses: pnpm/action-setup@a3252b78c470c02df07e9d59298aecedc3ccdd6d # v3.0.0
```

Comment:
question: why do we need this specific hash? Can't we use `version` option to use specific `pnpm` version and use latest GH action?

![CleanShot 2024-03-15 at 11 41 54@2x](https://github.com/saleor/macaw-ui/assets/4144459/6c2c3e28-af46-4d11-9808-ac2d08b059eb)

or it doesn't work for 3.x.x`? 😅

Reply:
This is pinning github action for PNPM not version of PNPM inself 😄 . We added it based on GitHub recommendation https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions

Reply:
Nice, I didn't know about this once ❤️